host/flush_tracer.go

```diff
@@ -18,11 +18,20 @@ import (
 
 type TraceEvent struct {
 	Type      string
+	Topic     string
```

Comment:
Is it necessary to have the topic field also in the Trace event? (doesn't hurt but feels duplicated)
The extended Payload should already have the Topic of the message. Are you guys going to need it for some easier filter afterwards?

Reply:
> Are you guys going to need it for some easier filter afterwards?

Yeah bang on! 💯 Xatu receives the `*host.TraceEvent` and uses the `Topic` to determine what the extended payload data is, to then cast into a more concrete type, eg: `TraceEventAttestation`, `TraceEventVoluntaryExit`, etc.

> doesn't hurt but feels duplicated

Agree, I was a bit hesitant to add it, but couldn't find another way that wasn't really intrusive.